src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -73,15 +85,14 @@ public SimpleQueryStringBuilder field(String field, float boost) {
         this.fields.put(field, boost);
         return this;
     }
-
+    
     /**
-     * Specify a name for the query
-     */
-    public SimpleQueryStringBuilder queryName(String name) {
-        this.queryName = name;
-        return this;
+     * Returns the fields including their respective boosts to run the query against.
+     * */
+    public Map<String, Float> fields() {
+        return Collections.unmodifiableMap(this.fields);
```

Comment:
not sure it is worth to make a copy for the getter, you can set fields through setter anyway no?